spec/graphql/queries/existing_database/get_one_drink_for_a_bar_spec.rb

```diff
@@ -72,5 +72,118 @@ def query_drink
     end
   end
 
+  describe 'sad path' do
+    it "gets an error if no drink ID is rec'd"  do
+      def query_bad_id_drink
+        <<~GQL
+          query {
+            drink(id: 300) {
+              id
+              name
+              imgUrl
+              steps
+              createdAt
+              updatedAt
+              bar {
+                id
+                name
+                drinkCount
+              }
+              ingredients {
+                name
+                quantity
+                createdAt
+                updatedAt
+              }
+            }
+          }
+        GQL
+      end
+
+      post '/graphql', params: {query: query_bad_id_drink}
+      result = JSON.parse(response.body)
+
+      expect(result["errors"][0]["message"]).to eq("Couldn't find Drink with 'id'=300")
+    end
+  end
+
+  describe 'edge case' do
+    it 'does not receive and error when a field is requested errently twice' do
```

Comment:
Check the description for this test I think there is an extra partial word.

Reply:
Should be fixed as requested, thanks for noticing!